fri/README.md

```diff
@@ -13,7 +13,7 @@ $ pip install jupyterlab
 
 To run the FRI as a server:
 ````
-$ cd conore
+$ cd conore/fri
```

Comment:
Yes, this was an issue in the dev branch. It was updated in the main branch. Now I have merged the pull request. So this README should be fixed now.